.github/CODEOWNERS

```
@@ -0,0 +1 @@
+* @observIQ/devops
```

```suggestion
* @observIQ/platform-tooling
```

devops is not a public team but 
platform-tooling is, same members.